bc/core/utils/status/templates.py

```diff
@@ -104,3 +109,43 @@
 
 #CL{docket_id}""",
 )
+
+
+BLUESKY_FOLLOW_A_NEW_CASE = TwitterTemplate(
+    link_placeholders=[],
+    str_template="""I'm now following {docket}:
+
+{docket_link}
+
+#CL{docket_id}""",
+)
```

Comment:
Shouldn't these use `BlueskyTemplate` instead of `TwitterTemplate`?

Reply:
You're right! I'm fixing this.